Formula/viewvc.rb

```diff
@@ -14,7 +14,7 @@ class Viewvc < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "19c07a79667814ccb1b14b6214a3d5fcca65ec31381e6e46a5db3ac3f72fc2d4"
   end
 
-  deprecate! date: "2022-10-27", because: "has no python 3 support"
+  disable! date: "2023-04-17", because: "has no python 3 support"
 
   depends_on :macos # Due to Python 2 (https://github.com/viewvc/viewvc/issues/138)
```

Comment:
I suggest making this `:maximum_macos` and let it die a slow death instead.

Reply:
Done